.github/workflows/release.yml

```diff
@@ -0,0 +1,56 @@
+name: "Manual Release workflow"
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: Release version
+        required: true
+
+jobs:
+  tagged-release:
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        java-version: [ 8 ]
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: master
+      - uses: actions/setup-java@v2
+        with:
+          distribution: 'temurin'
+          java-version: ${{ matrix.java-version }}
+          cache: 'maven'
+      - name: Configure Git user
+        run: |
+          git config user.email "actions@github.com"
+          git config user.name "GitHub Actions"
+          git remote -v
+      - name: Set version in env
+        run: |
+          version="${GITHUB_REF#refs/tags/}"
+          echo "RELEASE_VERSION=${{ github.event.inputs.version }}" >> $GITHUB_ENV
```

Comment:
The first line of this step is wrong now, since there's no tag. There should be no need for this step at all anymore, though, since the version is explicitly provided as input to the workflow.

Reply:
Removed this step and using `${{ github.event.inputs.version }}` wherever needed.